internal/plugins/linux/selinux_test.go

```diff
@@ -3,6 +3,7 @@
 //go:build linux
 // +build linux
 
+//nolint:revive
```

Comment:
What was this linter error about? Just asking out of curiosity, as I see it's disabled globally for the file.

Reply:
I've added this to ignore dot-imports lint error below
![image](https://github.com/user-attachments/assets/4f974afe-f7da-4c7e-bf74-dc67d83e9131)

Reply:
Thanks! Feel free to merge 🙂